src/interactive/accordionPanel.test.jsx

```diff
@@ -1,45 +1,64 @@
 import React from 'react';
 import { shallow, mount } from 'enzyme';
 
-import AccordionPanel, { ACTIVEPANEL_CLASS, ACCORDIONPANEL_CONTENT_CLASS } from './AccordionPanel';
+import AccordionPanel, {
+	ACTIVEPANEL_CLASS,
+	ACCORDIONPANEL_CONTENT_CLASS,
+} from './AccordionPanel';
 import Icon from '../media/Icon';
 import ToggleSwitch from '../forms/ToggleSwitch';
 
 describe('AccordionPanel', function() {
-	let panel,
-		openPanel;
+	let panel, openPanel;
 
 	const customIcon = 'plus';
 	const customIconActive = 'minus';
 
-	let panelStateCallback,
-		onClickCallback;
+	let panelStateCallback, onClickCallback;
 
 	beforeEach(() => {
 		panelStateCallback = jest.fn();
 		onClickCallback = jest.fn();
 
 		panel = shallow(
 			<AccordionPanel
-				label='First Section'
+				label="First Section"
 				setClickedPanel={panelStateCallback}
 				onClickCallback={onClickCallback}
 				panelContent={
-					<div className='runningText'>
-						<p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.</p>
+					<div className="runningText">
+						<p>
+							Contrary to popular belief, Lorem Ipsum is not simply random text.
```

Comment:
Same as above; it might be nice to have a separate file, like `textContentMocks`, to use for populating content in tests